tests/examples_methods_syntax/scatter_point_paths_hover.py

```diff
@@ -0,0 +1,95 @@
+"""
+Scatter plot with point paths on hover with search box
+======================================================
+This example demonstrates an interactive visualization technique inspired by [this Vega example](https://vega.github.io/vega/examples/global-development/), 
+adapted for Altair's higher-level API. Key features:
+
+1. Visual Hint Paths: Displays trajectories of data items across time on hover, enabling intuitive understanding of how data evolves.
+2. Search Box: Allows filtering of data points by country name, extending the original concept with improved data exploration capabilities
+
+"""
+# category: interactive charts
+
+import altair as alt
+from vega_datasets import data
+
+# Data source
+source = data.gapminder.url
+
+# X-value slider
+x_slider = alt.binding_range(min=1955, max=2005, step=5, name='Year: ')
+x_select = alt.selection_point(name="x_select", fields=['year'], bind=x_slider, value=1980)
+
+# Hover selection
+hover = alt.selection_point(on='mouseover', fields=['country'], empty=False)
+
+# Search box for country name
+search_box = alt.param(
+    value='',
+    bind=alt.binding(input='search', placeholder="Country", name='Search ')
+)
+
+# Base chart
+base = alt.Chart(source).encode(
+    x=alt.X('fertility:Q', scale=alt.Scale(zero=False), title='Fertility'),
+    y=alt.Y('life_expect:Q', scale=alt.Scale(zero=False), title='Life Expectancy'),
+    color=alt.Color('region_name:N').legend(title='Region'),
+    detail='country:N'
+).transform_calculate(
+    region_name="{'0': 'South Asia', '1': 'Europe & Central Asia', '2': 'Sub-Saharan Africa', '3': 'America', '4': 'East Asia & Pacific', '5': 'Middle East & North Africa'}[datum.cluster]"
```

Comment:
```suggestion
    color=alt.Color('region:N', title='Region'),
    detail='country:N'
).transform_calculate(
    region="""{
        '0': 'South Asia',
        '1': 'Europe & Central Asia',
        '2': 'Sub-Saharan Africa',
        '3': 'America',
        '4': 'East Asia & Pacific',
        '5': 'Middle East & North Africa'
    }[datum.cluster]"""

```

Setting title the same way as for x and y